### PR TITLE
[FCE-622] Karkakol/fix android e2e tests

### PR DESCRIPTION
**Pull request by karkakol**
## Description

Restore android e2e tests

### CODE REVIEW

File: examples/fishjam-chat/app.json [hunk 1]
```diff
@@ -45,7 +47,15 @@
           }
         }
       ],
-      ["./plugin/build/with-local-paths-for-native-packages.js"] // Only relevant for local development of packages/ios-client and packages/android-client
```
Comment (MiloszFilimowski) [inline examples/fishjam-chat/app.json:48]: **issue:** Why remove the comment? `app.json` is [JSON5](https://json5.org/) under the hood, so this is valid syntax.

File: examples/fishjam-chat/app.json [hunk 2]
```diff
@@ -45,7 +47,15 @@
...
+      ["./plugin/build/with-local-paths-for-native-packages.js"],
+      [
+        "expo-build-properties",
```
Comment (MiloszFilimowski) [inline examples/fishjam-chat/app.json:52]: **suggestion:** Can you move `expo-build-properties` before our plugins? Just to make sure we override everything as the last step.

File: examples/webdriverio-test/package.json [hunk 3]
```diff
@@ -24,6 +23,7 @@
     "webdriverio": "^9.1.1"
   },
   "dependencies": {
+    "@fishjam-cloud/js-server-sdk": "^0.1.2",
```
Comment (MiloszFilimowski) [inline examples/webdriverio-test/package.json:26]: **question:** Why this change? What does it do?

**Review by maksg — APPROVED** [review]

**Review by MiloszFilimowski — APPROVED** [review]

**Review by MiloszFilimowski — APPROVED** [review]
